deepchem/models/tensorgraph/symmetry_functions.py

```diff
@@ -448,3 +449,11 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
                                                              self.out_channels))
       outputs.append(output)
     self.out_tensor = tf.add_n(outputs)
+
+  def none_tensors(self):
```

Comment:
I had to implement none_tensors and set_tensors because the layer had class variables which were tf.tensor objects.